src/victory-util/helpers.js

```diff
@@ -1,8 +1,30 @@
 import React from "react";
-import { defaults, isFunction, property, omit, reduce } from "lodash";
+import { defaults, isFunction, property, reduce } from "lodash";
 import Collection from "./collection";
 
 export default {
+  /**
+   * creates an object with some keys excluded
+   * replacement for lodash.omit for performance. does not mimick the entire lodash.omit api
+   * @param {Object} originalObject: created object will be based on this object
+   * @param {Array<String>} keys: an array of keys to omit from the new object
+   * @returns {Object} new object with same properties as originalObject
+   */
+  omit(originalObject, keys = []) {
```

Comment:
Can you refactor this to **not** be part of the `default` export? 

We've got tickets already to unwind shoving everything in this mega object.

Then just switch to `import { omit } from "../victory-util/helpers";` and we're off!

Reply:
oh nice! yeah i'd prefer that anyway :)